openvdb_houdini/SOP_NodeVDB.h

```diff
@@ -61,6 +61,22 @@ class OPENVDB_HOUDINI_API OpenVDBOpFactory: public houdini_utils::OpFactory
     /// Construct an OpFactory that on destruction registers a new OpenVDB operator type.
     OpenVDBOpFactory(const std::string& english, OP_Constructor, houdini_utils::ParmList&,
         OP_OperatorTable&, houdini_utils::OpFactory::OpFlavor = SOP);
+
+    /// Register the node
+    ~OpenVDBOpFactory();
+
+    /// @brief Set the name of the parent operator.
+    /// @details This is typically the name of the operator shipped with Houdini and is
+    /// only needed where the parent name policy doesn't provide the correct name.
```

Comment:
Add something like "Pass an empty string to indicate that there is no equivalent native operator."